### ISSUE TITLE
Add gstreamer dlls to pyinstaller (Trac #112)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#116]
In order to get the pyinstaller-generated exe to run without gstreamer installed, we need to do the following:

1. Add gstreamer plugins to a sub-directory within the pyinstaller distribution directory (e.g., "dist/microdrop/gst_plugins")
2. Set the GST_PLUGIN_PATH environment variable to point to this sub-directory and also add this directory to system path before importing gst (this could be done after the import statements in microdrop.py).
3. Add contents of "%OSSBUILD_GSTREAMER_DIR%\bin" to the pyinstaller distribution directory (don't replace files that already exist).

Following these steps, I was able to successfully run the pyinstaller-generated exe without any gstreamer dlls in my path. Note that not all of the included files may be necessary, but seeing as it's only ~40Mb, it's probably not a problem to just add them all for the time being.

Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/112
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "In order to get the pyinstaller-generated exe to run without gstreamer installed, we need to do the following:\n\n1. Add gstreamer plugins to a sub-directory within the pyinstaller distribution directory (e.g., \"dist/microdrop/gst_plugins\")\n2. Set the GST_PLUGIN_PATH environment variable to point to this sub-directory and also add this directory to system path before importing gst (this could be done after the import statements in microdrop.py).\n3. Add contents of \"%OSSBUILD_GSTREAMER_DIR%\\bin\" to the pyinstaller distribution directory (don't replace files that already exist).\n\nFollowing these steps, I was able to successfully run the pyinstaller-generated exe without any gstreamer dlls in my path. Note that not all of the included files may be necessary, but seeing as it's only ~40Mb, it's probably not a problem to just add them all for the time being.", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Add gstreamer dlls to pyinstaller", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-07-14T15:48:51", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120728T02:47:43: 
* ryan@fobel.net commented:

> commit 8c2f79a697df76803dc016c130086cd2f86977a9
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Fri Jul 27 22:16:41 2012 -0400
> 
>     Fix installer/gstreamer dll issues
>     
>     Create a new binary dependencies repository. Fixes #115, #116.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "" to "microdrop"
